adds extra cmds to fc to table

shostetter opened the pull request:
This allows user to pass in additional gdal commands  to the end of the existing command, to allow for greater flexibility

shostetter committed:
adds extra cmds to fc to table

cindyatnycdot reviewed (approved):
I wasn't able to get some of the tests to pass, but I think it's my computer and how it uploads/formats geometries. I don't think it's a big deal since we're changing tests/functions in my `geospatial` branch soon and I believe I changed the tests in my branch already.

Tests that weren't passing for me: 
`TestFeatureClassToTablePg::test_import_fc_new_name_schema_srid`
`TestFeatureClassToTablePg::test_import_fc_new_name_data_check`

Very small thing, but your PR didn't have any updates to the Ms tests, only the Pg tests.

shostetter committed:
update bulk loader to remove the internal flag

this was an error that lead to tables being imported via bulk not being in the tables created list

shostetter commented:
> I wasn't able to get some of the tests to pass, but I think it's my computer and how it uploads/formats geometries. I don't think it's a big deal since we're changing tests/functions in my `geospatial` branch soon and I believe I changed the tests in my branch already.
> 
> Tests that weren't passing for me: `TestFeatureClassToTablePg::test_import_fc_new_name_schema_srid` `TestFeatureClassToTablePg::test_import_fc_new_name_data_check`
> 
> Very small thing, but your PR didn't have any updates to the Ms tests, only the Pg tests.

yes because like ii said this is incomplete because it will be replaced very soon by your geospatial redesign - can you make sure you add this feature to all of the GDAL imports/exports in your code?
